aas-web-ui/src/components/SubmodelPlugins/_SubmodelEntrypoint.vue

```diff
@@ -48,6 +48,14 @@
                     checkSemanticId(submodelElementData, 'https://admin-shell.io/ZVEI/TechnicalData/Submodel/1/2')
                 "
                 :submodel-element-data="submodelElementData"></TechnicalData>
+            <PCF
+                v-else-if="
+                    checkSemanticId(
+                        'https://admin-shell.io/idta/CarbonFootprint/CarbonFootprint/0/9',
+                        submodelElementData
```

Comment:
Please swap the two parameter of checkSemanticId()
See [SubmodelElementHandling.ts](https://github.com/eclipse-basyx/basyx-aas-web-ui/blob/0f2532f17f36c4c8d74d93c69b79d595cdc38faa/aas-web-ui/src/mixins/SubmodelElementHandling.ts#L109C9-L109C24): `checkSemanticId(submodelElement: any, semanticId: string)`